compiler/rustc_codegen_llvm/src/coverageinfo/map_data.rs

```diff
@@ -254,7 +244,30 @@ impl<'tcx> FunctionCoverage<'tcx> {
 struct ZeroExpressions(FxIndexSet<ExpressionId>);
 
 impl ZeroExpressions {
+    fn new() -> Self {
+        Self(FxIndexSet::default())
+    }
+
+    fn insert(&mut self, id: ExpressionId) {
```

Comment:
I don't see any uses of this method.

Reply:
It's used inside `identify_zero_expressions`, as a drop-in replacement for `FxIndexSet::insert`.

Previously that method was building up an `FxIndexSet` directly (with direct calls to `FxIndexSet::insert` and `FxIndexSet::contains`), and wrapping it in `ZeroExpressions` at the end. Now it creates an empty `ZeroExpressions` and inserts entries via the wrapper, so that it can call the unified helper function that takes `&ZeroExpressions`.